11ty/definitions/colonialism.md

```diff
@@ -1,5 +1,27 @@
 ---
-title: Colonialism
+title: colonialism
```

Comment:
To prevent messy PRs, will you please delete this file in the PR? That way we are addressing any of its edits in its own PR (which you've already done).

This PR should only update neo-colonialism. Thank you!

Reply:
@tatianamac this is done! Also did it while listening to your talk :) Always fabulous!

Reply:
😭😭😭 You're too kind to me!!!